pkg/ddl/index_presplit.go

```diff
@@ -120,6 +118,17 @@ func getSplitIdxKeysFromValueList(
 		return getSplitIdxPhysicalKeysFromValueList(sctx, tblInfo, idxInfo, tblInfo.ID, byRows, destKeys)
 	}
 
+	if idxInfo.Global {
+		var idxTblID int64
+		if pi.NewTableID != 0 && idxInfo.State != model.StatePublic {
```

Comment:
Please add comment for this

Reply:
This is used in reorg partition. Since we don't support pre_split option in reorg partition, let's remove this.